Makefile

```diff
@@ -16,6 +16,9 @@ test-unit:
 test-system: build
 	@go test -v -tags=system ./tests -args PATH=$(shell pwd)/dist
 
+test-docs: build
```

Comment:
New make target to run the docs test. I had to split them out due to their zq-sample-data requirement. Of course if necessary we can modify the CI setup to clone that repo and run the docs test as part of every run.